specifications/FiniteMonotonic/MC_CRDT.tla

```diff
@@ -0,0 +1,72 @@
+------------------------------- MODULE MC_CRDT ------------------------------
```

Comment:
Perhaps, choose a different name to emphasize its importance, and not just yet another "dump" MC.tla.